app/assets/javascripts/react_on_rails.js

```diff
@@ -156,10 +100,59 @@
     return consoleReplay;
   };
 
-  function createReactElement(componentName, propsVarName, props, domId, trace, generatorFunction) {
+  function forEachComponent(fn) {
+    var els = document.getElementsByClassName('react-component');
```

Comment:
@justin808 oh sorry I thought you were talking about the function name, now I got ya!